src/lib/experiments/tests/liveblog-right-column-ads.ts

```diff
@@ -0,0 +1,36 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const liveblogRightColumnAds: ABTest = {
+	id: 'LiveblogRightColumnAds',
+	author: '@commercial-dev',
+	start: '2023-07-15',
+	expiry: '2023-09-20',
+	audience: 0 / 100,
+	audienceOffset: 0 / 100,
+	audienceCriteria: 'Desktop users with wide (1300px+) screens only',
```

Comment:
Do you want to use the `canRun` predicate below to constrain to an eligible set of pageviews? E.g. desktop liveblogs or something like that.

Reply:
Ah yes good spot thanks! We also don't need to check for the showcase element here, as that isn't used on liveblogs

Reply:
Hmm it might be more effort than it's worth, and easier to just filter pageviews in the analysis.